errors.go

```diff
@@ -44,6 +44,9 @@ var (
 	// ErrWorkspaceMinLimit is returned when the length of Workspaces is 0.
 	ErrWorkspaceMinLimit = errors.New("must provide at least one workspace")
 
+	// ErrMissingTagIdentifier is returned if a Tag argmunet lacks an Id or Name to match on
+	ErrMissingTagIdentifier = errors.New("bad request\n\nMust specify at least one tag")
```

Comment:
Nit: no other errors have a header like this
```suggestion
	ErrMissingTagIdentifier = errors.New("must specify at least one tag")
```

Reply:
i thought that was strange too but that's what came back from the API in the tests.

Reply:
You don't need to match that. The format you see is [the client itself interpreting a response from the platform, using the response code as a header.](https://github.com/hashicorp/go-tfe/blob/0743349bef2e7efb1ce3666321c718e6fa9dbd28/tfe.go#L744) This is a client-side validation. 

No biggie, we'll amend it before releasing.